app/controllers/LilaController.scala

```diff
@@ -31,7 +31,7 @@ abstract private[controllers] class LilaController(val env: Env)
 
   implicit protected val LilaResultZero = Zero.instance[Result](Results.NotFound)
 
-  implicit final protected class LilaPimpedResult(result: Result) {
+  implicit final protected class LilaLilaResult(result: Result) {
```

Comment:
I think one Lila should be enough, no? 😄

Reply:
One would be forgiven for thinking this...  but in fact I have discovered a new formulation of the Lambda Calculus using search/replace operator and the `Lila` string, and this is historical work.